### PR TITLE
added README for 1st workshop day

### PR DESCRIPTION
**Pull request by woutslakhorst**
Also changed request URLs to use address from workshop context.

### CODE REVIEW

File: network_in_a_day/README.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+## Start a network
+
+We'll use the network setup as defined by the **nuts-network-local** repository.
+First clone the github repo to your machine:
+
+```shell
+git clone https://github.com/nuts-foundation/nuts-network-local
+```
+
+If you already did this as part of the preparation, please pull the latest changes.
```
Comment (reinkrul) [inline network_in_a_day/README.md:52]: And pull the latest versions of the docker images;

`docker-compose pull`

File: network_in_a_day/README.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+Inside the repository there are 2 directories: *single* and *network*. We'll use the setup from the **single** directory.
+Inside that directory startup the docker containers:
+
+```shell
+docker-compose up
+```
+
+To check if everything is running well, follow the [diagnostics manual](../mini-manuals/1-node-diagnostics.md).
+Next we'll join all nodes to form a network.
```
Comment (reinkrul) [inline network_in_a_day/README.md:62]: It's not necessary to have all nodes connect to all nodes, I think?

Maybe we should instruct participants to enter their IP addresses in a google sheets or some other place which everyone can view and edit?

File: network_in_a_day/README.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+  bootstrapnodes:
+    - 192.168.1.xxx:5555
+    - 192.168.1.yyy:5555 
+```
+
+where `xxx` and `yyy` are to replaced by you peers IPs.
+
+Stop the docker containers (if not done already) and start them again:
+```shell
+docker-compose down
```
Comment (reinkrul) [inline network_in_a_day/README.md:83]: `docker-compose restart`

**Review by reinkrul — CHANGES_REQUESTED** [review]

File: network_in_a_day/README.md (hunk 2 above)
Comment (woutslakhorst) [reply network_in_a_day/README.md:62]: Maybe there will be paper signs (for the name) that we can use.

**Pull request merged**